src/apps/core/app.js

```diff
@@ -89,13 +90,35 @@ define(function(require) {
 			self.displayFavicon();
 			self.loadSVG();
 
+			if (crossSiteMessaging !== undefined) {
+				self.bindCrossSiteMessagingHandler(crossSiteMessaging);
+			}
+
 			container.append(mainTemplate);
 
 			monster.waterfall([
 				_.bind(self.loadIncludes, self)
 			], _.bind(self.loadAuth, self));
 		},
 
+		bindCrossSiteMessagingHandler: function(crossSiteMessaging) {
+			const { origin, topics } = crossSiteMessaging;
+
+			window.addEventListener('message', handler);
+
+			function handler(event) {
```

Comment:
Declare the function before it is added as an event listener, using a more explicit name, to follow Monster UI conventions. For example:
```suggestion
			var handleCrossSiteMessages = function handleCrossSiteMessages(event) {
```